aspnet/fundamentals/servers/sample/ServersDemo/src/ServersDemo/Startup.cs

```diff
@@ -1,55 +1,47 @@
+using Microsoft.AspNetCore.Hosting.Server.Features;
+using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.PlatformAbstractions;
-using Microsoft.Net.Http.Server;
-using System;
-using System.Linq;
-
 
 namespace ServersDemo
 {
     public class Startup
     {
-
-        public Startup(IHostingEnvironment env, 
-            IApplicationEnvironment appEnv)
-        {
-            // Setup configuration sources.
-            var configBuilder = new ConfigurationBuilder()
-                .SetBasePath(appEnv.ApplicationBasePath)
-                .AddEnvironmentVariables();
-
-            Configuration = configBuilder.Build();
-
-            
-        }
-        public IConfiguration Configuration { get; private set; }
-        public void ConfigureServices(IServiceCollection services)
+        public Startup(IHostingEnvironment env)
         {
+            Configuration = new ConfigurationBuilder()
+                .SetBasePath(env.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+                .AddEnvironmentVariables()
+                .Build();
         }
 
-        public void Configure(IApplicationBuilder app, IApplicationLifetime lifetime, ILoggerFactory loggerFactory)
+        public IConfigurationRoot Configuration { get; private set; }
+
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
-            var webListenerInfo = app.ServerFeatures.Get<WebListener>();
-            if (webListenerInfo != null)
-            {
-                webListenerInfo.AuthenticationManager.AuthenticationSchemes =
-                    AuthenticationSchemes.AllowAnonymous;
-            }
+            loggerFactory.AddConsole(Configuration.GetSection("Logging"));
 
-            var serverAddress = app.ServerFeatures.Get<IServerAddressesFeature>()?.Addresses.FirstOrDefault();
+            var serverAddressesFeature = app.ServerFeatures.Get<IServerAddressesFeature>();
 
+            app.UseStaticFiles();
             app.Run(async (context) =>
             {
-                var message = String.Format("Hello World from {0}",
-                                        serverAddress);
-                await context.Response.WriteAsync(message);
+                await context.Response.WriteAsync($"Hosted by {Program.Server}\r\n\r\n");
+
+                if (serverAddressesFeature != null)
+                {
+                    await context.Response.WriteAsync($"Listening on the following addresses: {string.Join(", ", serverAddressesFeature.Addresses)}\r\n");
+                }
+
+                await context.Response.WriteAsync($"Local connection info: {context.Connection.LocalIpAddress}:{context.Connection.LocalPort}\r\n");
```

Comment:
Correct. I'm open to any way to write this that's clean enough for sample code. Maybe just remove those two lines?